packages/core/core/src/Transformation.js

```diff
@@ -333,29 +329,26 @@ export default class Transformation {
   }
 
   async getPipelineHash(pipeline: Pipeline): Promise<string> {
-    let hash = crypto.createHash('md5');
+    let hashes = '';
     for (let transformer of pipeline.transformers) {
       let key = `${transformer.name}:${transformer.resolveFrom}`;
-      hash.update(
+      hashes +=
         this.request.devDeps.get(key) ??
-          this.devDepRequests.get(key)?.hash ??
-          '',
-      );
+        this.devDepRequests.get(key)?.hash ??
+        '';
```

Comment:
In theory, the same thing could happen here?

Reply:
There will always be at least on dev dep - the plugin itself. I think the `?? ''` is just there to appease flow.

Reply:
Should it be `nullthrows` instead of `??` then?

Reply:
Actually seems I was wrong. It will be null on initial builds.